pluto.jl

```diff
@@ -250,22 +250,26 @@ end
 
 # ╔═╡ 7b979410-8af0-11eb-299c-af0a5d740c24
 md"""
-We can see that, despite an `AlignmentTest` only having 4 + 2 + 1 = 7 bytes of actual data, it takes up 8 bytes of memory, and accessing an `AlignmentTest` object from an array will always be aligned.
+我们可以看到，尽管 `AlignmentTest` 只有 4 + 2 + 1 = 7 字节的真实数据，但是却分配了 8 字节的内存。
+这样的话，访问数组中的 `AlignmentTest` 对象时，内存仍然是对齐的。
 
-As a coder, there are only a few situations where you can face alignment issues. I can come up with two:
+作为一名程序员，其实只有极少数情况会遇到内存对齐问题。这里提供两个例子：
 
-1. If you manually create an object with a strange size, e.g. by accessing a dense integer array with pointers. This can save memory, but will waste time. [My implementation of a Cuckoo filter](https://github.com/jakobnissen/Probably.jl) does this to save space.
-2. During matrix operations. If you have a matrix the columns are sometimes unaligned because it is stored densely in memory. E.g. in a 15x15 matrix of `Float32`s, only the first column is aligned, all the others are not. This can have serious effects when doing matrix operations: [I've seen benchmarks](https://juliasimd.github.io/LoopVectorization.jl/latest/examples/matrix_vector_ops/) where an 80x80 matrix/vector multiplication is 2x faster than a 79x79 one due to alignment issues.
+1. 若创建的对象具有奇怪的尺寸，例如使用指针访问稠密的整数数组。这种操作虽然可以节约内存，但是会浪费时间。我实现的[Cuckoo filter](https://github.com/jakobnissen/Probably.jl) 就使用了这种方式节约空间。 
+2. 矩阵操作的过程。因为数组元素在内存中紧靠着存储，所以有时数组的列会是未对齐的。例如，在一个 15x15 的 `Float32` 矩阵中，仅仅只有第一列是对齐的，其他列都存在未对齐问题。这会对矩阵操作造成严重的后果：[在链接里的基准测试中](https://juliasimd.github.io/LoopVectorization.jl/latest/examples/matrix_vector_ops/)，由于对齐问题，80x80 矩阵/向量的乘法会比 79x79 矩阵/向量的乘法快上两倍。
```

Comment:
单看中文好像不是很清楚 "紧靠着" 是什么意思

```suggestion
2. 矩阵操作的过程。因为数组元素在内存中是连续存储的，所以有时数组的列会是未对齐的。例如，在一个 15x15 的 `Float32` 矩阵中，仅仅只有第一列是对齐的，其他列都存在未对齐问题。这会对矩阵操作造成严重的后果：[在链接里的基准测试中](https://juliasimd.github.io/LoopVectorization.jl/latest/examples/matrix_vector_ops/)，由于对齐问题，80x80 矩阵/向量的乘法会比 79x79 矩阵/向量的乘法快上两倍。
```